common/lib/dependabot/config/file.rb

```diff
@@ -60,7 +60,8 @@ def ignore_conditions(cfg)
         ignores.map do |ic|
           Dependabot::Config::UpdateConfig::IgnoreCondition.new(
             name: ic[:"dependency-name"],
-            versions: ic[:versions]
+            versions: ic[:versions],
+            update_type: ic[:"update-type"]
```

Comment:
I think this will end up being an array
```suggestion
            update_types: ic[:"update-types"]
```